.github/workflows/testing.yml

```
@@ -0,0 +1,263 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  detect-changes:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read # Only needs to read repository contents
+    outputs:
+      packages: ${{ steps.packages.outputs.packages }}
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)